### ISSUE TITLE
kdb spec-mount error prints wrong keyname

### ISSUE THREAD

**Issue by kodebach** [ElektraInitiative/libelektra#3665]
Points: 1

## Steps to Reproduce the Problem

If `spec:/tests/example` doesn't exist

```sh
kdb meta-get spec:/tests/example mountpoint
#> Key not found

kdb spec-mount spec:/tests/example
```

Alternatively, if only `mountpoint` the metakey doesn't exist:

```sh
kdb meta-get spec:/tests/example mountpoint
#> Metakey not found

kdb spec-mount spec:/tests/example
```

## Expected Result

`kdb` prints an error that the metakey `mountpoint` wasn't found:

```
No metadata "mountpoint" found on key "spec:/tests/example"
```

## Actual Result

`kdb` prints:

```
No metadata "mountpoint" found below spec/tests/example
```

## System Information

- Elektra Version: master

**Comment by markus2330** [comment]
Points: 1

The problem in src/tools/kdb/specmount.cpp line 86

**Comment by a-kraschitzer** [comment]
I would like to take this over.

**Comment by mpranj** [comment]
Closed via #3790 (origin: #3775) thanks to @a-kraschitzer!
